code/modules/mob/living/carbon/carbon.dm

```diff
@@ -615,10 +615,12 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	return TRUE
 
 /mob/living/carbon/restrained()
-	if(handcuffed)
+	if(get_retraining_item())
 		return TRUE
-	return
+	return FALSE
 
+/mob/living/carbon/get_retraining_item()
```

Comment:
typo? you spelled re training not re**s**training

Reply:
Will change once able. Good spot

Reply:
will merge once this typo is fixed

Reply:
Typo is fixed